compiler/src/dmd/expressionsem.d

```diff
@@ -653,7 +653,7 @@ private Expression resolveUFCS(Scope* sc, CallExp ce)
     if (!ce.names)
         ce.names = new Identifiers();
     ce.names.shift(null);
-
+    ce.isUfcsRewrite = true;
```

Comment:
Can this not be inferred either during semantic analysis (less state, less memory)?

Reply:
This comment was meant to be attached to the new bool

Reply:
This PR doesn't increase memory usage of `CallExp` since it fits in a padding byte. When you want to add another `bool`, the 4 existing bools can be packed into bit fields. 

I don't see how `expressionSem(CallExp)` can infer that the `CallExp` came from UFCS, unless it's stored in there.